src/mixin-props.js

```diff
@@ -56,8 +56,26 @@ module.exports = function mixinDefine(Base = HTMLElement) {
 			installEventTarget(this.constructor);
 		}
 
-		intialize(props) {
-			super.intialize(props);
+		initialize(props) {
+			super.initialize(props);
+			let prop, staticProps;
+
+			if (this.constructor.props) {
+				staticProps = Object.keys(this.constructor.props);
+			}
+
+			for (prop in this) {
+				if (this.hasOwnProperty(prop)) {
+					if (staticProps && staticProps.includes(prop)) {
+						const val = this[prop];
+						delete this[prop];
```

Comment:
Why do you need to delete the prop?

Reply:
Because `this[prop]` will take the class field property with the same name.
`this.constructor.props[prop]` is not settable.